### PR TITLE
[13.0] [FIX] account_invoice_fiscal_position_update: Filter only lines without comments or notes.

### PR DESCRIPTION
**Pull request by nicomacr**
Here if you create a note or a comment in the invoice lines, and you change the fpos an error occurs.
After this changes only non display type lines are taking into account.

### CODE REVIEW

**Comment by nicomacr** [comment]
ping @OCA/accounting-maintainers

**Review by jjscarafia — APPROVED** [review]
LGTM

**Comment by PabloPaezSheridan** [comment]
Hi @rvalyi, can you view these changes and approved if it is okey? is important to us. Thanks

**Review by alexis-via — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please fix pre-commit

**Comment by nicomacr** [comment]
@pedrobaeza pre-commit ready!

**Pull request merged**
